schema/crdb/route-config-rib-priority/up02.sql

```diff
@@ -0,0 +1,9 @@
+SET LOCAL disallow_full_table_scans = off;
+UPDATE omicron.public.switch_port_settings_route_config
+SET rib_priority = 
+    CASE 
+        WHEN local_pref > 255 THEN 255
+        WHEN local_pref < 0 THEN 0
+        ELSE local_pref::INT2
+    END;
+SET LOCAL disallow_full_table_scans = on;
```

Comment:
I don't think you need to turn this back on; each step of the migration is run within its own transaction.